myhpi/core/templates/core/minutes.html

```diff
@@ -8,69 +8,69 @@
     <div class="row minutes-container">
         {% with page.body|markdown as parsed_md %}
         <div class="d-none d-print-block minutes-title">
-        <h1>{{  page.title }}</h1>
+            <h1>{{  page.title }}</h1>
         </div>
         <div class="col-9 minutes-text">
             {{ parsed_md.0 }}
         </div>
         <div class="col-3 minutes-meta">
-            <h4>{% translate "Date" %}</h4>
-            <p>{{ page.date }}</p>
-            <h4>{% translate "Moderator" %}</h4>
-            <p>{{ page.moderator.get_full_name }}</p>
-            <h4>{% translate "Minutes taker" %}</h4>
-            <p>{{ page.author.get_full_name }}</p>
-            <h4>{% translate "Participants" %}</h4>
-            <ul>
-                {% for participant in page.participants.all %}
-                    <li>{{ participant.get_full_name }}</li>
-                {% endfor %}
-            </ul>
-            <h4>{% translate "Guests" %}</h4>
-            {% if page.guests %}
+            <aside class="side-panel border-accent">
+                <h1 class="toc-title">{% translate "Date" %}</h1>
```

Comment:
This class seems to be misnamed now. Maybe side-panel-title?

Reply:
Fixed.